src/pages/Script/index.tsx

```diff
@@ -153,12 +153,12 @@ export const ScriptPage = () => {
         <HashCardPanel isColumn={false}>
           <ScriptsTitleOverview scriptInfo={scriptInfo} />
         </HashCardPanel>
-        <Tabs
+        <ScriptTab
```

Comment:
The close button has disappeared, which may be unexpected.

Reply:
done

Reply:
This size is not quite reasonable.

![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/9160743/f7716b47-2b5b-4f30-96a6-43b5db5fbae5)